package/MDAnalysis/analysis/msd.py

```diff
@@ -301,7 +304,7 @@ class EinsteinMSD(AnalysisBase):
     .. versionadded:: 2.0.0
     """
 
-    def __init__(self, u, select="all", msd_type="xyz", fft=True, **kwargs):
+    def __init__(self, u, select="all", msd_type="xyz", fft=True, non_linear = False, **kwargs):
```

Comment:
Document new argument. Should look something like
```reST
non_linear : bool
    (describe what the option enables)

    .. versionadded:: 2.10.0

```
(Make sure that there's space under the versionadded and *visually check the built docs* that it's formatted correctly – under Checks there's the Read The Docs build that has the built docs)

Reply:
Also add an entry to CHANGELOG under Enhancements documenting the new features including the new kwarg.

Reply:
Run black over your code, I am pretty sure that it should not have spaces around `=` in argument lists.